scripts/zones/Promyvion-Vahzl/npcs/qm_wailer.lua

```diff
@@ -19,8 +19,4 @@ entity.onTrade = function(player, npc, trade)
     end
 end
 
-entity.onTrigger = function(player, npc)
```

Comment:
These QMs have no message? or is IF handling it and i don't see it in this PR

Reply:
I'm adding a couple of new entries to the DefaultActions.lua for these messages. Provoker QM already had the message there.